pkgs/applications/audio/jack-autoconnect/default.nix

```
@@ -0,0 +1,38 @@
+# Call with libsForQt5.callPackage
+
+{ lib, mkDerivation, fetchFromGitHub, pkg-config, qmake, qtbase, libjack2 }:
+
+let
+  # 1 February 2021
+  gitRevision = "fe0c8f69149e30979e067646f80b9d326341c02b";
+
+in mkDerivation rec {
+  pname = "jack_autoconnect";
+
+  # It does not have any versions (yet?)
+  version = "git-${builtins.substring 0 7 gitRevision}";
```

Please follow [the package version name guidelines](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming) for this case:

> If a package is not a release but a commit from a repository, then the `version` attribute *must* be the date of that (fetched) commit. The date *must* be in `"unstable-YYYY-MM-DD"` format.

```suggestion
  version = "unstable-2021-02-01";
```

---

Force-pushed the update. Also updated MR title.